### DISCUSSION TITLE
blog/github-action-timeout

### DISCUSSION THREAD

**Discussion by giscus** [AshikNesin/nesinio-comments#17] [Announcements]
# blog/github-action-timeout

https://nesin.io/blog/github-action-timeout

**Comment by vitalykarasik** [comment]
>"And we can configure it at workflow level or even at the step level as well."  

I guess you mean "on the *job* or step level".

**Comment by ldomesjo** [comment]
I believe that your steps-example is missing an "s" at the end, should be timeout-minutes both for jobs and job steps
